src/waveview/wave.py

```diff
@@ -13,7 +13,7 @@
 class RootWave(BoxLayout):
     sample_rate = 44100
     num_samples = 44100
-    time = 5
+    time = 2
```

Comment:
Why 2s? Just for now to reduce delay?

Reply:
I was trying to fix the x-coordinates of the waveform graph, but now that I think of it, this doesn't fix it.

Reply:
Is it broken atm?

Reply:
Not broken but wrong

Reply:
ok. maybe merge this PR. Cause I think all the other changes are good. And then make another ticket for investigating why it's wrong

Reply:
And I think 2s is actually good cause that's the time period we agreed on mapping (with you being able to change the period to the next 2s)